core/spec/lib/spree/core/testing_support/factories/order_factory_spec.rb

```diff
@@ -45,4 +45,10 @@
 
     it_behaves_like 'a working factory'
   end
+
+  describe 'completed order with promotion' do
+    let(:factory) { :completed_order_with_promotion }
+
+    it_behaves_like 'a working factory'
+  end
```

Comment:
So we agreed we do want to have better specs for factories for sure. @graygilmore Just give us a heads-up if you're willing to spec out this factory a little more; @jhawthorn has given a fine example in #1264 on how to do this beautifully.